packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -295,6 +296,7 @@ export default class SortedTable extends Component {
         sortCriteria: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
         sortOrder: PropTypes.string,
         textAlign: PropTypes.string,
+        valuePath: PropTypes.string,
```

Comment:
Group `component`, `render` and `valuePath` and add a comment explaining how and when to use them.